### PR TITLE
periodic cleanup of registered (and no longer valid) servers on qe sprayproxy

### PR DESCRIPTION
**Pull request by xinredhat**
This PR depends on PR https://github.com/redhat-appstudio/e2e-tests/pull/899. It will periodically   call the `clean-registered-servers` to clean up the invalid pac servers registered in QE Sprayproxy.

### CODE REVIEW

File: ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh [hunk 1]
```diff
@@ -47,7 +47,8 @@ make clean-github-webhooks || CLEAN_WEBHOOK_STATUS=$?
 make clean-quay-repos-and-robots || CLEAN_QUAY_REPOS_AND_ROBOTS_STATUS=$?
 make clean-quay-tags || CLEAN_QUAY_TAGS_STATUS=$?
 make clean-private-repos || CLEAN_PRIVATE_REPO_STATUS=$?
+make clean-registered-servers || CLEAN_PRIVATE_REPO_STATUS=$?
```
Comment (tisutisu) [inline ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh:50]: Will it be good to capture `make clean-registered-servers` status to another variable like `CLEAN_REGISTERED_SERVERS_STATUS` and use it in the next step to check if it is `-ne 0`?

Comment (xinredhat) [reply ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh]: ah, my bad, thank you for catching this issue. fixed.

**Comment by tisutisu** [comment]
/pj-rehearse periodic-ci-redhat-appstudio-e2e-tests-main-appstudio-dump-external-resources

**Comment by xinredhat** [comment]
/pj-rehearse periodic-ci-redhat-appstudio-e2e-tests-main-appstudio-dump-external-resources

**Comment by xinredhat** [comment]
/pj-rehearse periodic-ci-redhat-appstudio-e2e-tests-main-appstudio-dump-external-resources

**Review by rsoaresd — APPROVED** [review]

**Review by kasemAlem — APPROVED** [review]

**Pull request merged**